packages/containerd/containerd-config-toml_k8s_nvidia

```diff
@@ -0,0 +1,39 @@
+version = 2
+root = "/var/lib/containerd"
+state = "/run/containerd"
+disabled_plugins = [
+    "io.containerd.internal.v1.opt",
+    "io.containerd.snapshotter.v1.aufs",
+    "io.containerd.snapshotter.v1.devmapper",
+    "io.containerd.snapshotter.v1.native",
+    "io.containerd.snapshotter.v1.zfs",
+]
+
+[grpc]
+address = "/run/dockershim.sock"
+
+[plugins."io.containerd.grpc.v1.cri"]
+enable_selinux = true
+# Pause container image is specified here, shares the same image as kubelet's pod-infra-container-image
+sandbox_image = "{{settings.kubernetes.pod-infra-container-image}}"
+
+[plugins."io.containerd.grpc.v1.cri".containerd]
+default_runtime_name = "nvidia"
```

Comment:
Why are we setting this as the default?  Wouldn't the typical experience be to have it available as an option set through [RuntimeClass](https://kubernetes.io/docs/concepts/containers/runtime-class/)?

Reply:
The typical experience with EKS Optimzed AMI, CO'OS, and the GPU Operator is that the users don't have to set a `RuntimeClass` to run their workloads. This settings helps us have parity with at least the EKS Optimized AMI.